cartographer_ros/cartographer_ros/occupancy_grid_node_main.cc

```diff
@@ -40,6 +40,10 @@
 DEFINE_double(resolution, 0.05,
               "Resolution of a grid cell in the published occupancy grid.");
 DEFINE_double(publish_period_sec, 1.0, "OccupancyGrid publishing period.");
+DEFINE_bool(ignore_frozen_submaps, false,
```

Comment:
Perhaps it is less negations and easier to read with "show_(un)frozen_submaps" and default true?